doc/reference/configuration/configuration_reference.rst

```diff
@@ -3220,6 +3220,26 @@ The ``lua`` section outlines the configuration parameters related to the Lua env
     | Default: 2147483648 (2GB)
     | Environment variable: TT_LUA_MEMORY
 
+.. _configuration_reference_lua_call:
+
+.. confval:: lua_call
```

Comment:
Is it on the top level? The deployment fails, so I can't check how it is rendered.

Reply:
Seems solved.